app_dart/lib/src/request_handlers/push_build_status_to_github.dart

```diff
@@ -43,22 +43,38 @@ class PushBuildStatusToGithub extends ApiRequestHandler<Body> {
   final BuildStatusServiceProvider buildStatusServiceProvider;
   final HttpClientProvider branchHttpClientProvider;
   final GitHubBackoffCalculator gitHubBackoffCalculator;
+  static const String repoParam = 'repo';
 
   @override
   Future<Body> get() async {
+    final String repo = request.uri.queryParameters[repoParam] ?? 'flutter';
     final Logging log = loggingProvider();
     final DatastoreService datastore = datastoreProvider(config.db);
     final BuildStatusService buildStatusService = buildStatusServiceProvider(datastore);
-    final RepositorySlug slug = config.flutterSlug;
-    final GithubService githubService = await config.createGithubService(slug);
+    // TODO(godofredoc): find a better way to create the map of repo to repository slug.
+    final Map<String, RepositorySlug> slugMap = <String, RepositorySlug>{
```

Comment:
If you pass the full repo slug, you can do `Repo.fromFullName`

Reply:
Not sure what Repo are you referring to I could find any reference to either Repo or fromFullName.

Reply:
done